src/app/contact/contact.component.html

```diff
@@ -46,7 +46,7 @@ <h2 class="bold">Contact Us</h2>
     <div class="row">
       <div class="col-lg-12 col-sm-12 sub-details">
         <h5 class="bold">Susper</h5>
-        <p>93 Mau Than Street<br> Can Tho<br> Viet Nam<br> Phone +84 (0) 907 65 29 27<br><a href="mailto: support@susper.net"> Email: support@susper.net</a><br>Board
+        <a href="https://www.openstreetmap.org/search?query=93%20Mau%20Than%20Street%2C%20Can%20Tho#map=19/10.03117/105.77529"><p>93 Mau Than Street<br> Can Tho<br> Viet Nam<br></a> Phone +84 (0) 907 65 29 27<br><a href="mailto: support@susper.net"> Email: support@susper.net</a><br>Board
```

Comment:
@saarthakchats You start an <a> tag and then inside, you start a `<p>` tag. However, you then close the `<a>` tag before you close the `<p>` tag. This is causing the issues. I think you can fix the issue by starting the `<p>` tag before the `<a>` tag.